tests/test_dynamics_bookmarks.py

```diff
@@ -0,0 +1,213 @@
+import datetime
+import dateutil.parser
+import pytz
+
+from tap_tester import runner, menagerie, connections
+
+from base import DynamicsBaseTest
+
+
+class DynamicsBookmarks(DynamicsBaseTest):
+    @staticmethod
+    def name():
+        return "tap_tester_dynamics_bookmarks"
+
+    def get_properties(self, original: bool = True):
```

Comment:
I think this method can be dropped here.

Reply:
removed